release_notes/ocp-4-13-release-notes.adoc

```diff
@@ -74,10 +74,13 @@ With this release, {product-title} {product-version} introduces a {op-system-bas
 ==== VMware vSphere region and zone enablement
 You can deploy an {product-title} cluster to multiple vSphere datacenters or regions that run in a single VMware vCenter. Each datacenter can run multiple clusters or zones. This configuration reduces the risk of a hardware failure or network outage causing your cluster to fail.
 
-You can also specify multiple regions and zones on a cluster after installation. You must upgrade your existing cluster to {product-title} {product-version} before you configure a multiple region and zone environment.
+[IMPORTANT]
+====
+For {product-title} {product-version}, you cannot deploy an upgraded cluster to multiple vSphere regions in a VMware vCenter instance. You must install a cluster on {product-title} {product-version}, configure the multiple vSphere datacenter environment, and then deploy the cluster in this environment. 
```

Comment:
I'd say something along these lines: "Only newly installed clusters can use this feature . You cannot use the new region and zones introduced in this release in a cluster upgraded from a previous release since this feature depends on using the CSI StorageClass, which isn't enabled in upgraded clusters."